eva-stats/stats_variant_warehouse.py

```diff
@@ -70,8 +75,14 @@ def get_from_variant_warehouse(mongo_handle, metadata_handle, projects):
             assembly, project, analysis, filename = row[0], row[1], row[2], row[4]
             analyses.append(analysis)
             analysis_filename[analysis] = filename
-        get_counts_from_variant_warehouse(assembly, project, analyses, metadata_handle, mongo_handle)
-        get_dates_from_variant_warehouse(assembly, project, analysis_filename, metadata_handle, mongo_handle)
+        try:
+            database_name = get_database_name_from_assembly(metadata_handle, assembly)
+            print(database_name)
```

Comment:
logger.info here?

Reply:
Changed, it was supposed to be a `logger.info`. Thanks